packages/app/src/hooks/useContextMenuAddNodeConfiguration.ts

```diff
@@ -59,7 +59,7 @@ export const addContextMenuGroups = [
 };
 
 export function useContextMenuAddNodeConfiguration() {
-  const constructors = globalRivetNodeRegistry.getNodeConstructors();
+  const constructors = useMemo(() => globalRivetNodeRegistry.getNodeConstructors(), []);
```

Comment:
I think this might need to depend on the plugins enabled, because the node constructors change based on what nodes are available and registered

Reply:
Thanks, you are correct.
I didn't check the code in core package.